mypy_django_plugin/main.py

```diff
@@ -52,25 +51,39 @@ def add_new_manager_base(ctx: ClassDefContext) -> None:
 
 
 def extract_django_settings_module(config_file_path: Optional[str]) -> str:
-    errors = Errors()
-    if config_file_path is None:
-        errors.report(0, None, "'django_settings_module' is not set: no mypy config file specified")
-        errors.raise_error()
+
+    def exit(error_type: int) -> NoReturn:
+        """Using mypy's argument parser, raise `SystemExit` to fail hard if validation fails.
+
+        Considering that the plugin's startup duration is around double as long as mypy's, this aims to
+        import and construct objects only when that's required - which happens once and terminates the
+        run. Considering that most of the runs are successful, there's no need for this to linger in the
+        global scope.
+        """
+        from mypy.main import CapturableArgumentParser
+
+        usage = '\n'.join(['(config)',
+                           '...',
+                           '[mypy.plugins.django_stubs]',
```

Comment:
Just realised that this entire nonsense is a constant, and that the tests already define a string for it:
```
TEMPLATE = """usage: (config)
...
[mypy.plugins.django_stubs]
    django_settings_module: str (required)
...
(django-stubs) mypy: error: 'django_settings_module' is not set: {}
"""
```

Assuming that we follow DRY, and considering that error handling is about to get extended in general, I feel like most of this `exit` function should move somewhere else, say, a module called `errors`? It could live in `mypy_django_plugin/lib/` like `helpers` and `fullnames`. I thought of adding it to the helpers, but it somehow seems wrong to `from helpers import exit` which kills the app.